asv_bench/benchmarks/groupby.py

```diff
@@ -710,63 +641,31 @@ def time_transform_multi_key4(self):
         self.df4.groupby(['jim', 'joe'])['jolie'].transform('max')
 
 
+class TransformBools(object):
 
-
-np.random.seed(0)
-N = 120000
-N_TRANSITIONS = 1400
-transition_points = np.random.permutation(np.arange(N))[:N_TRANSITIONS]
-transition_points.sort()
-transitions = np.zeros((N,), dtype=np.bool)
-transitions[transition_points] = True
-g = transitions.cumsum()
-df = DataFrame({'signal': np.random.rand(N), })
-
-
-
-
-
-class groupby_transform_series(object):
     goal_time = 0.2
 
     def setup(self):
-        np.random.seed(0)
         N = 120000
         transition_points = np.sort(np.random.choice(np.arange(N), 1400))
-        transitions = np.zeros((N,), dtype=np.bool)
+        transitions = np.zeros(N, dtype=np.bool)
         transitions[transition_points] = True
         self.g = transitions.cumsum()
         self.df = DataFrame({'signal': np.random.rand(N)})
 
-    def time_groupby_transform_series(self):
+    def time_transform_mean(self):
         self.df['signal'].groupby(self.g).transform(np.mean)
 
 
-class groupby_transform_series2(object):
+class TransformNaN(object):
+    # GH 12737
     goal_time = 0.2
 
     def setup(self):
-        np.random.seed(0)
-        self.df = DataFrame({'key': (np.arange(100000) // 3),
-                             'val': np.random.randn(100000)})
-
-        self.df_nans = pd.DataFrame({'key': np.repeat(np.arange(1000), 10),
-                                     'B': np.nan,
-                                     'C': np.nan})
-        self.df_nans.ix[4::10, 'B':'C'] = 5
-
-    def time_transform_series2(self):
-        self.df.groupby('key')['val'].transform(np.mean)
-
-    def time_cumprod(self):
-        self.df.groupby('key').cumprod()
-
-    def time_cumsum(self):
-        self.df.groupby('key').cumsum()
-
-    def time_shift(self):
```

Comment:
Where are those moved?

Reply:
These are essentially identical to the benchmarks in `GroupByMethods`

Reply:
ok